src/Raven.Server/Documents/Replication/ConflictManager.cs

```diff
@@ -137,6 +138,7 @@ public static void AssertChangeVectorNotNull(string conflictedChangeVector)
                 InvalidConflictWhenThereIsNone(conflict.Id);
 
             conflictedDocs.Add(conflict.Clone());
+            conflictedDocs = conflictedDocs.OrderByDescending(x => x.ChangeVector.ToChangeVectorList().Select(cv => cv.GetHashCode()).OrderByDescending(k => k).GetEnumerableHashCode()).ToList();
```

Comment:
No, you need to sort it by the string of the change vector, not this. I'm not sure how this will sort at all.

Reply:
This is when we resolve by script, we need to get same order of conflicted docs on all nodes/dbs, I used the hash of CV because it is the same for `A:1, B:1` and `B:1, A:1`, then ordered by `EnumerableHash`.

Reply:
You are calling GetHashCode, which isn't guaranteed to be identical
Note that the change vector _has_ to be the same across nodes.
There is no way you can get `A:1, B1` and `B:1, A1` in our system. The node that generated this decided on the format, and all else will follow that

Reply:
@ayende you can have the situation of `A:1, B1` and `B:1, A1` and we consider this as an identical change vector.
Consider two nodes send simultaneously and each of them resolve it independently.  

But I agree that `GetHashCode` on a string is not deterministic, I would say order them lexicographically by the dbId.

Reply:
You cannot get that. Only one server can generate a change vector, and the version it generates is what it will use.

Reply:
changed to sort on string, and fixed the situation of `A:1, B1` and `B:1, A1` in cv